### ISSUE TITLE
🚀 Merged to dev: SymptomSelection did not render

### ISSUE THREAD

**Issue by fauortiz** [umed-jp/umed#53]
New user got stuck on the symptoms selection screen due to the symptom buttons not rendering.

<img width="460" alt="Screenshot_2024-04-02_at_10 06 29" src="https://github.com/umed-jp/umed/assets/67045972/71ef83dc-c61a-4a00-989b-b650734d153f">

**Comment by fauortiz** [comment]
I can't seem to reproduce the bug. I tried the following attempts to reproduce the bug:
- fresh account & existing account
- new email
- testflight & dev
- no Patient role
- network disconnection

The closest I got was my very first time using testflight, using my email with a deleted account. While the medical interview was loading in, the symptoms were empty, like the bug. The buttons rendered after loading finished though. Every time after this though, the symptoms were never empty even when loading the page.

Current hypothesis is it failed to fetch `settings` from firestore. Currently investigating what could cause it.
